### ISSUE TITLE
minified code can not produce fp64_ref result

### ISSUE THREAD

**Issue by desertfire** [pytorch/pytorch#93458]
Minified code:
```
import torch._inductor.overrides
import torch
from torch import tensor, device
import torch.fx as fx
from torch._dynamo.testing import rand_strided
from math import inf
from torch.fx.experimental.proxy_tensor import make_fx
# torch version: 1.14.0a0+fb
# torch cuda version: 11.4.0
# CUDA Info: 
# nvcc not found
# GPU Hardware Info: 
# NVIDIA A100-PG509-200 : 8 
from torch.nn import *
class Repro(torch.nn.Module):
    def __init__(self):
        super().__init__()


    def forward(self, arg147_1, arg149_1, arg151_1, arg153_1, add_37, permute_62, permute_63, permute_64, permute_65):
        convert_element_type_11 = torch.ops.prims.convert_element_type.default(add_37, torch.float32);  add_37 = None
        addmm_60 = torch.ops.aten.addmm.default(arg147_1, convert_element_type_11, permute_62);  arg147_1 = permute_62 = None
        addmm_61 = torch.ops.aten.addmm.default(arg149_1, addmm_60, permute_63);  arg149_1 = addmm_60 = permute_63 = None
        mul_54 = torch.ops.aten.mul.Tensor(convert_element_type_11, addmm_61);  addmm_61 = None
        add_38 = torch.ops.aten.add.Tensor(mul_54, convert_element_type_11);  mul_54 = convert_element_type_11 = None
        addmm_62 = torch.ops.aten.addmm.default(arg151_1, add_38, permute_64);  arg151_1 = add_38 = permute_64 = None
        addmm_63 = torch.ops.aten.addmm.default(arg153_1, addmm_62, permute_65);  arg153_1 = addmm_62 = permute_65 = None
        return (addmm_63,)

args = [((256,), (1,), torch.float32, 'cuda'), ((7616,), (1,), torch.float32, 'cuda'), ((256,), (1,), torch.float32, 'cuda'), ((7616,), (1,), torch.float32, 'cuda'), ((8, 7616), (7616, 1), torch.float32, 'cuda'), ((7616, 256), (1, 7616), torch.float32, 'cuda'), ((256, 7616), (1, 256), torch.float32, 'cuda'), ((7616, 256), (1, 7616), torch.float32, 'cuda'), ((256, 7616), (1, 256), torch.float32, 'cuda')]
args = [rand_strided(sh, st, dt, dev) for (sh, st, dt, dev) in args]
mod = make_fx(Repro().to(device="cuda"))(*args)
from torch._inductor.compile_fx import compile_fx_inner
from torch._dynamo.debug_utils import same_two_models
compiled = compile_fx_inner(mod, args)
assert same_two_models(mod, compiled, args, only_fwd=True), "Accuracy failed"                                                                            
```

Warning:
```
[WARNING] Could not generate fp64 outputs
```

The problem is likely coming from `torch.ops.prims.convert_element_type.default(add_37, torch.float32)` and it is not a bug, but not being able to compare with `fp64_ref` sometimes can lead minifier to a wrong direction when minifying accuracy errors.
